akka-actor-typed-tests/src/test/scala/akka/actor/typed/WidenSpec.scala

```diff
@@ -36,11 +46,14 @@ class WidenSpec extends ScalaTestWithActorTestKit with WordSpecLike {
       val probe = TestProbe[String]()
       val ref = spawn(intToString(probe.ref))
 
-      ref ! 42
-      ref ! 13
-      ref ! 43
-      probe.expectMessage("42")
-      probe.expectMessage("43")
+      // non-convertible/filtered message 13 is logged as warning
```

Comment:
ok, should it? I thought it was a feature of `widen` to be able to filter, or why is it a partial function? change to debug logging?

Reply:
I thought this was something already present in master and just wanted to silence it, but now that I double check that the warning log entry is actually introduced by this PR. I'll look into how that happened.

Reply:
Yeah, Chris is right, it is because the TestEventListener for some reason logs `unhandled` messages as `warning`. I'll update the comment and keep the silencer.